test/TestUtil.js

```diff
@@ -654,6 +680,16 @@ exports.calculateServerVersionFromString = (versionString) => {
  * This function will wait for the connections count to be equal to given parameter (connectionCount).
  */
 exports.waitForConnectionCount = async (client, connectionCount) => {
+    const getConnectionsFn = await exports.getClientConnections(client);
+    await this.assertTrueEventually(async () => {
+        expect(getConnectionsFn().length).to.be.equal(connectionCount);
+    });
+};
+
+/**
+ * Returns client getConnectionsFn according to client version
+ */
+exports.getClientConnections = async (client) => {
```

Comment:
I think we should rename this function to "getClientConnectionsFn"

Reply:
Fixed